doc/Homepage.ipynb

```diff
@@ -17,7 +17,7 @@
    "source": [
     "import numpy as np\n",
     "import holoviews as hv\n",
-    "%reload_ext holoviews.ipython\n",
+    "hv.notebook_extension('matplotlib')\n",
```

Comment:
Thanks for raising that; I meant to list it as an issue.  Yes, the reason I put 'matplotlib' explicitly, here on the homepage is to make it clear that (a) we are a way to access plotting libraries, not competing with them, (b) we support multiple backends, not just one, and (c) to imply that a user could type 'bokeh' here to switch the backend.  I'd rather avoid putting the comment explicitly, to keep the homepage as simple as possible; I'm pretty sure that if a reader looks at the rest of the page and sees that we support matplotlib and Bokeh, they will guess to try 'bokeh' or 'Bokeh' here, and of course they can also link on the Bokeh tutorials to verify that.